.ddev/config.yaml

```diff
@@ -1,17 +1,17 @@
 name: akela
 type: php
 docroot: public
-php_version: "8.2"
+php_version: "8.3"
 webserver_type: nginx-fpm
 xdebug_enabled: false
-additional_hostnames: []
-additional_fqdns: []
+additional_hostnames: [ ]
+additional_fqdns: [ ]
 database:
-    type: mariadb
```

Comment:
non so come mai ma mi ha messo i rientri dello yaml a 2 spazi anzichè a 4... Devo sistemare qualcosa in PHPStorm?  🤷

Reply:
si, ti ha aggiunto anche altri spazi 🤔 
magari fai il revert delle righe tranne quella di php 8.3